tests/vmi_lifecycle_test.go

```diff
@@ -205,6 +206,39 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			Expect(pod.Annotations).To(HaveKey("kubernetes.io/test"), "kubernetes annotation should not be carried to the pod")
 		})
 
+		It("Should prevent eviction when EvictionStratgy: External", func() {
```

Comment:
we're only interested in ensuring that the VMI is marked with the EvacuationNodeName when this occurs so the external controller can manage the situation.

All we're guaranteed with this new feature is that KubeVirt will provide the signal `vmi.Status.EvacuationNodeName` when eviction is blocked. It's up to the external controller to determian what that means... it might mean shutdown, it might mean something else.